src/components/Navigation/TopBar.tsx

```diff
@@ -32,6 +34,7 @@ function TopBar({breadcrumbLabel, shouldDisplaySearch = true, shouldDisplayHelpB
     const [session] = useOnyx(ONYXKEYS.SESSION, {selector: authTokenTypeSelector, canBeMissing: true});
     const shouldShowLoadingBarForReports = useLoadingBarVisibility();
     const isAnonymousUser = isAnonymousUserUtil(session);
+    const isRhpVisible = useRootNavigationState((state) => state?.routes?.at(-1)?.name === NAVIGATORS.RIGHT_MODAL_NAVIGATOR);
```

Comment:
It does not look correct, as RHP can have any screen. Do we have to hide the top bar? Did we confirm this with the product team? cc: @mountiny 

https://github.com/user-attachments/assets/bd6e1fbf-9dd8-4926-bbf3-f301a3f43ad5